components/helptext/index.css

```diff
@@ -11,99 +11,87 @@
-		--spectrum-helptext-top-to-workflow-icon: var(--spectrum-help-text-top-to-workflow-icon-extra-large);
-		--spectrum-helptext-bottom-to-workflow-icon: var(--spectrum-helptext-top-to-workflow-icon);
+		--spectrum-helptext-edge-to-workflow-icon: var(--spectrum-component-top-to-workflow-icon-200);
 		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-200);
 		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-200);
 	}
 
+	&.spectrum-HelpText--sizeXL {
+		--spectrum-helptext-min-height: var(--spectrum-component-height-300);
+		--spectrum-helptext-icon-size: var(--spectrum-workflow-icon-size-300);
+		--spectrum-helptext-font-size: var(--spectrum-font-size-300);
+		--spectrum-helptext-text-to-visual: var(--spectrum-text-to-visual-300);
+		--spectrum-helptext-edge-to-workflow-icon: var(--spectrum-component-top-to-workflow-icon-300);
+		--spectrum-helptext-top-to-text: var(--spectrum-component-top-to-text-300);
+		--spectrum-helptext-bottom-to-text: var(--spectrum-component-bottom-to-text-300);
+	}
+
 	&.spectrum-HelpText--neutral {
 		--spectrum-helptext-content-color-default: var(--spectrum-neutral-subdued-content-color-default);
 		--spectrum-helptext-icon-color-default: var(--spectrum-neutral-subdued-content-color-default);
 	}
 
 	&.spectrum-HelpText--negative {
-		--spectrum-helptext-content-color-default: var(--spectrum-negative-color-900);
-		--spectrum-helptext-icon-color-default: var(--spectrum-negative-color-900);
+		--spectrum-helptext-content-color-default: var(--spectrum-negative-content-color-default);
+		--spectrum-helptext-icon-color-default: var(--spectrum-negative-content-color-default);
 	}
 
 	&.is-disabled {
 		--spectrum-helptext-content-color-default: var(--spectrum-disabled-content-color);
 		--spectrum-helptext-icon-color-default: var(--spectrum-disabled-content-color);
 	}
+}
 
+.spectrum-HelpText {
 	color: var(--highcontrast-helptext-content-color-default, var(--mod-helptext-content-color-default, var(--spectrum-helptext-content-color-default)));
 	display: flex;
 	font-size: var(--mod-helptext-font-size, var(--spectrum-helptext-font-size));
 	min-block-size: var(--mod-helptext-min-height, var(--spectrum-helptext-min-height));
 
-	&:lang(ja),
-	&:lang(zh),
-	&:lang(ko) {
-		--mod-helptext-line-height: var(--mod-helptext-line-height-cjk, var(--spectrum-cjk-line-height-100));
-	}
-
 	.spectrum-HelpText-validationIcon {
 		margin-inline-end: var(--mod-helptext-text-to-visual, var(--spectrum-helptext-text-to-visual));
-		padding-block-start: var(--mod-helptext-top-to-workflow-icon, var(--spectrum-helptext-top-to-workflow-icon));
-		padding-block-end: var(--mod-helptext-bottom-to-workflow-icon, var(--spectrum-helptext-bottom-to-workflow-icon));
+		padding-block-start: var(--mod-helptext-edge-to-workflow-icon, var(--spectrum-helptext-edge-to-workflow-icon));
+		padding-block-end: var(--mod-helptext-edge-to-workflow-icon, var(--spectrum-helptext-edge-to-workflow-icon));
```

Comment:
Personally, I think I like this better than keeping top and bottom separate variables! Pointing out though that we do lose the ability to modify different spacings for top vs. bottom. Given that design specs often point out when it's really important that consumers be able to modify something, and it's not here, I think using a single variable is ok, but pointing this out anyway in case someone has a stronger opinion about it.